javascript/ql/lib/semmle/javascript/frameworks/CookieLibraries.qll

```diff
@@ -132,6 +148,10 @@ private module JsCookie {
     }
 
     override predicate isSensitive() { canHaveSensitiveCookie(this.getArgument(0)) }
+
+    override string getSameSite() {
+      this.getOptionArgument(2, "sameSite").mayHaveStringValue(result)
```

Comment:
Should we introduce a shared (inline) predicate for this string/bool -> string conversion? I list the suggestion here because I do not see the harm in adding it for this smaller predicate as well.